client/store/modules/repository/comments/getters.js

```diff
@@ -5,3 +5,9 @@ export const getActivityComments = state => activityId => {
   const activityComments = filter(state.items, { activityId });
   return orderBy(activityComments, 'createdAt', 'desc');
 };
+
+export const getUnseenComments = state => (comments, activityUid) => {
```

Comment:
Make sure that comments are not created by the current user (If a current user posted the last comment then it makes no sense to show him the notification).